x-pack/packetbeat/Jenkinsfile.yml

```diff
@@ -39,6 +39,9 @@ stages:
     unitTest:
         mage: "mage build unitTest"
         stage: mandatory
+    systemTest:
+        mage: "mage testSystem"
```

Comment:
Can this be renamed to `systemTest`?

Reply:
It could; it's a pre-existing function that I have extended. I don't know how much it gets used though.